src/mqtt/MQTT.cpp

```diff
@@ -62,12 +62,17 @@ MQTT::MQTT() : concurrency::OSThread("mqtt"), pubSub(mqttClient)
 void MQTT::reconnect()
 {
     // pubSub.setServer("devsrv.ezdevice.net", 1883); or 192.168.10.188
-    const char *serverAddr = "mqtt.meshtastic.org"; // default hostname
+    const char *serverAddr = "mqtt.meshtastic.org:1883"; // default hostname
 
     if (*radioConfig.preferences.mqtt_server)
         serverAddr = radioConfig.preferences.mqtt_server; // Override the default
 
-    pubSub.setServer(serverAddr, 1883);
+    String server = String(serverAddr);  
+    int delimIndex = server.indexOf(':'); 
+    String host = server.substring(0, delimIndex);
+    String port = server.substring(delimIndex+1, server.length());
```

Comment:
This algorithm will fail when `serverAddr` does not contain ':'.  The default port should be 1883 if delilmIndex isn't valid.

Reply:
Good call. My initial thought was that we would just document the format, but this will be more user friendly.